### PR TITLE
migrate volume/csi/csi_attacher.go logs to structured logging

### PR DESCRIPTION
**Pull request by CKchen0726**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
part of kubernetes/enhancements#1602

Which issue(s) this PR fixes:
keps/sig-instrumentation/1602-structured-logging

Structured Logging migration instructions

Special notes for your reviewer:

Does this PR introduce a user-facing change?

```release-notes
migrate volume/csi/csi_attacher.go logs to structured logging
```

### CODE REVIEW

File: pkg/volume/csi/csi_attacher.go [hunk 1]
```diff
@@ -236,36 +235,36 @@ func (c *csiAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.No
 				attached[spec] = attach.Status.Attached
 				continue
 			}
-			klog.V(4).Info(log("attacher.VolumesAreAttached failed in AttachmentLister for attach.ID=%v: %v. Probing the API server.", attachID, err))
+			klog.V(4).ErrorS(err, log("attacher.VolumesAreAttached failed in AttachmentLister for attachment. Probing the API server"), "attachment", attachID)
```
Comment (yangjunmyfm192085) [inline pkg/volume/csi/csi_attacher.go:238]: Info - InfoS?

File: pkg/volume/csi/csi_attacher.go [hunk 2]
```diff
@@ -119,9 +119,9 @@ func (c *csiAttacher) Attach(spec *volume.Spec, nodeName types.NodeName) (string
 			if !apierrors.IsAlreadyExists(err) {
 				return "", errors.New(log("attacher.Attach failed: %v", err))
 			}
-			klog.V(4).Info(log("attachment [%v] for volume [%v] already exists (will not be recreated)", attachID, pvSrc.VolumeHandle))
```
Comment (yangjunmyfm192085) [inline pkg/volume/csi/csi_attacher.go:122]: I am not sure log format use “ migrate” can work here?

**Comment by humblec** [comment]
@CKchen0726 can you please revisit this PR ?

**Pull request closed without merging**